backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/chipgroup/multiple/internal/BpkMultiSelectChipGroupImpl.kt

```diff
@@ -0,0 +1,190 @@
+import net.skyscanner.backpack.compose.chip.internal.BpkDismissibleChipImpl
+import net.skyscanner.backpack.compose.chipgroup.multiple.BpkMultiChipGroupType
+import net.skyscanner.backpack.compose.chipgroup.multiple.BpkMultiChipItem
+import net.skyscanner.backpack.compose.divider.BpkDivider
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+internal fun BpkMultiSelectChipGroupImpl(
+    chips: List<BpkMultiChipItem>,
+    type: BpkMultiChipGroupType,
+    style: BpkChipStyle,
+    modifier: Modifier = Modifier,
+) {
+    when (type) {
+        is BpkMultiChipGroupType.Rail -> {
+            Row(modifier = modifier) {
+                type.stickyChip?.let { chip ->
+                    StickyChip(chip = chip, style = style)
+                }
+                LazyRow(
+                    modifier = Modifier.selectableGroup(),
+                    state = rememberLazyListState(),
+                ) {
+                    items(items = chips) { chip ->
+                        ChipItem(
+                            chip = chip,
+                            style = style,
+                            modifier = Modifier
+                                .padding(PaddingValues(BpkSpacing.Sm))
+                                .semantics { role = Role.Checkbox },)
+                    }
+                }
+            }
+        }
+
+        BpkMultiChipGroupType.Wrap -> {
+            FlowRow(
+                modifier = modifier.selectableGroup(),
+            ) {
+                chips.forEach { chip ->
+                    ChipItem(
+                        chip = chip,
+                        style = style,
+                        modifier = Modifier
+                            .padding(PaddingValues(BpkSpacing.Sm))
+                            .semantics { role = Role.Checkbox },)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun StickyChip(
+    chip: BpkMultiChipItem,
+    style: BpkChipStyle,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+            .padding(PaddingValues(BpkSpacing.Sm))
+            .height(IntrinsicSize.Min),
+    ) {
+        StickyChipItem(
+            chip = chip,
+            style = style,
+            modifier = Modifier
+                .padding(PaddingValues(end = BpkSpacing.Md))
+                .semantics {
+                    role = Role.Button
+                    contentDescription = chip.text
+                },
+        )
+        BpkDivider(
+            modifier = Modifier
+                .fillMaxHeight()
+                .width(1.dp),
+
+        )
+    }
+}
+
+@Composable
+private fun StickyChipItem(
+    chip: BpkMultiChipItem,
+    style: BpkChipStyle,
+    modifier: Modifier = Modifier,
+) {
+    val interactionSource = remember { MutableInteractionSource() }
+    when (chip.type) {
+        BpkChipType.Selectable, BpkChipType.Dropdown -> BpkChipImpl(
```

Comment:
i would actually say let's set the type to `Selectable` for sticky chip items. if we ever support tablets properly that means we can maintain the same API as right now and add support to it, but on mobile dropdown + dismissible makes no sense for the filter one (and dismissible in general tbf)

Reply:
> i would actually say let's set the type to Selectable for sticky chip items. if we ever support tablets properly that means we can maintain the same API as right now and add support to it

Do you mean we should remove the when case statement and  just set the type to be always `Selectable` and ignore Sticky chip item given type?  

> but on mobile dropdown + dismissible makes no sense for the filter one (and dismissible in general tbf)

Yep I agree, there is no use case I can think of for dismissible chips in chip group.  maybe we should use a different enum class that has only selectable and drop down types and map them to the internal types instead?

Reply:
Yeah thinking about it it probably makes sense to have a separate data class for the sticky chip which doesn't have a type (we can always add it later on). Internally we can map it to the other type